src/shared/components/TopTabs/TopTabs.module.scss

```
@@ -54,6 +55,7 @@
 
   &__label {
     @include m.text-style-extended('lg', 500, text-default);
+    color: var(--text-secondary);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)